style.css

```diff
@@ -98,3 +101,343 @@ body {
   width: 20px;
   height: 20px;
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Please remove media queries for now. You are expected to build this page mobile-first, which means all your mobile styling will be in the CSS file before you later write desktop styling in media queries. So just remove the media queries and move the styling in them outside the queries.

Reply:
Issue resolved. Thank you